phpBB/styles/prosilver/template/posting_attach_body.html

```diff
@@ -32,8 +32,8 @@
 						<th class="attach-status">{L_PLUPLOAD_STATUS}</th>
 					</tr>
 				</thead>
-				<tbody class="responsive-skip-empty" id="file-list">
-					<tr class="attach-row" id="attach-row-tpl">
+				<tbody class="responsive-skip-empty file-list" id="file-list">
+					<tr class="attach-row attach-row-tpl" id="attach-row-tpl">
```

Comment:
```display: none``` for this is not overriden by plupload right now. Attachments do not show after upload in the filelist. They properly show after clicking on preview.